src/Mod/Sketcher/App/SketchAnalysis.cpp

```diff
@@ -373,7 +373,7 @@ void SketchAnalysis::makeMissingPointOnPointCoincident(bool onebyone)
             }
 
             if(status) {
-                THROWMT(Base::RuntimeError, QT_TRANSLATE_NOOP("Exceptions", "Autoconstrain error: Unsolvable sketch while applying coincident constraints.\n"))
+                THROWMT(Base::RuntimeError, QT_TRANSLATE_NOOP("Exceptions", "Autoconstrain error: Unsolvable sketch while applying coincident constraints.")) << "\n";
```

Comment:
There is no QString here or stream to use `<<` so not sure how to append newline to this...please advise

Reply:
@wmayer are newlines even necessary here?

Reply:
Need help with this :point_up:

Reply:
Actually the newline can be omitted here

Reply:
Newslines ommitted in 4cc7006